packages/cyberstorm/src/components/componentStyles/Button.module.css

```diff
@@ -0,0 +1,97 @@
+.button {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex: none;
+  gap: var(--gap-medium);
+
+  padding: var(--padding-medium-small) var(--padding-medium);
+  border-radius: var(--size-small);
+
+  background-clip: padding-box, border-box;
+  --bg-color: var(--color-surface--3);
+  background-color: var(--bg-color);
+  --text-color: var(--color-white);
+  color: var(--text-color);
+}
+
+.button__default {
+  --text-color: var(--color-white);
+  --bg-color: var(--color-surface--3);
+}
+.button__default:hover {
+  --bg-color: var(--color-surface--4);
+  outline: 2px solid var(--color-border-2);
+}
+
+.button__defaultDark {
+  --text-color: var(--color-white);
+  --bg-color: var(--color-surface--1);
+}
+
+.button__defaultDark:hover {
+  --bg-color: var(--color-surface--2);
+  outline: 2px solid var(--color-border-2);
+}
+
+.button__primary {
+  --text-color: var(--color-black);
+  --bg-color: var(--color-cyber-green);
+}
+
+.button__primary:hover {
+  --bg-color: var(--color-cyber-green--40);
+}
+
+.button__danger {
+  --text-color: var(--color-white);
+  --bg-color: var(--color-red);
+}
+
+.button__danger:hover {
+  --bg-color: var(--color-red--40);
+}
+
+.button__defaultWithBorder {
+  --bg-color: var(--color-surface--3);
+  --text-color: var(--color-white);
+  outline: 2px solid var(--color-border-2);
+}
+
+.button__defaultWithBorder:hover {
+  --bg-color: var(--color-surface--4);
+  outline: 2px solid var(--color-border-3);
+}
+
+.button__specialGreen {
+  --text-color: var(--color-white);
+  background-image: var(--color-gradient-blue-green--darker),
+  var(--color-gradient-blue-green);
+  border: 2px solid;
+  border-image-source: var(--color-gradient-blue-green);
+  box-shadow: 0 0 var(--size-medium) 0 var(--color-cyber-green--20);
+}
+
+.button__specialGreen:hover {
+  background-image: var(--color-gradient-blue-green--dark),
```

Comment:
I don't think this is animatable as is. I created a [Codepen entry](https://codepen.io/gravija/pen/poKzGYJ) to demonstrate how we could achieve smooth transition with gradients, which basically involves animating the opacity of a pseudo element. Let me know what do you think.

Reply:
I'll look into this

Reply:
Fixed using the example